src/gui/uithememanager.cpp

```diff
@@ -133,7 +132,15 @@ QIcon UIThemeManager::getIcon(const QString &iconId, const QString &fallback) co
 QIcon UIThemeManager::getFlagIcon(const QString &countryIsoCode) const
 {
     if (countryIsoCode.isEmpty()) return {};
-    return QIcon(":/icons/flags/" + countryIsoCode.toLower() + ".svg");
+
+    const QString key = countryIsoCode.toLower();
+    const auto iter = m_flagCache.find(key);
+    if (iter != m_flagCache.end())
+        return *iter;
+
+    const QIcon icon{QLatin1String(":/icons/flags/") + key + QLatin1String(".svg")};
```

Comment:
Coding Style is broken here.
Also PR has conflicts with master.

Reply:
rebased and fixed coding guideline